opennms-services/src/test/java/org/opennms/netmgt/collectd/SnmpCollectorWithMibPropertiesIT.java

```diff
@@ -242,6 +242,52 @@ public void testCollectCiscoQoS() throws Exception {
         assertEquals("GESTION-ROUTING", map.get("cbQosClassMapName"));
         assertEquals("Conexion Valencia", map.get("ifAlias"));
     }
+    
+    /**
+     * Test collection for Cisco memory-pool entry with indirectly-referenced property
+     * pulled over from entPhysicalTable
+     * 
+     * @throwsException the exception
+     */
+    @Test
+    @JUnitCollector(datacollectionType = "snmp", datacollectionConfig = "/org/opennms/netmgt/config/datacollection-config-cisco-mempool.xml")
+    @JUnitSnmpAgent(resource = "/org/opennms/netmgt/snmp/cisco-mempool-snmpwalk.properties")
+    public void testCollectionCiscoMemPoolVsEntPhysical() throws Exception {
+        System.setProperty("org.opennms.netmgt.collectd.SnmpCollector.limitCollectionToInstances", "true");
+        
+        CollectionSet collectionSet = m_collectionSpecification.collect(m_collectionAgent);
+        assertEquals("collection status", CollectionStatus.SUCCEEDED, collectionSet.getStatus());
+        CollectorTestUtils.persistCollectionSet(m_rrdStrategy, m_resourceStorageDao, m_collectionSpecification, collectionSet);
+        
+        Map<String, String> map = m_resourceStorageDao.getStringAttributes(ResourcePath.get("snmp", "1", "cempMemoryPool", "1.1"));
+        assertEquals("Processor", map.get("cempMemoryPoolName"));
+        assertEquals("CISCO2911/K9", map.get("cempMemoryPoolPhysName"));
+        assertEquals("CISCO2911/K9 chassis, Hw Serial#: FCZ161870SC, Hw Revision: 1.0", map.get("cempMemoryPoolPhysDescr"));
+        
+        map = m_resourceStorageDao.getStringAttributes(ResourcePath.get("snmp", "1", "cempMemoryPool", "1.2"));
+        assertEquals("CISCO2911/K9", map.get("cempMemoryPoolPhysName"));
+        assertEquals("CISCO2911/K9 chassis, Hw Serial#: FCZ161870SC, Hw Revision: 1.0", map.get("cempMemoryPoolPhysDescr"));
+    }
+
+    /**
+     * Test enum-lookup property extender against values of dot1dStpPortState
+     *
+     * @throwsException the exception
```

Comment:
```suggestion
     * @throws Exception the exception
```
Elsewhere there is a space between @throws and Exception. Not sure if this is syntactically necessary.